packages/paste-libraries/styling/src/index.tsx

```diff
@@ -0,0 +1,14 @@
+// Base styling system (emotion)
+import styled from '@emotion/styled';
+import * as emotion from 'emotion';
+
+export {Global as StylingGlobals, SerializedStyles, ThemeContext, keyframes} from '@emotion/core';
+export {ThemeProvider} from 'emotion-theming';
```

Comment:
`withTheme` is missing from here and is causing the theme package size to increase and some of the tests to fail. Not all tests